src/python_testing/TC_RVCCLEANM_2_2.py

```diff
@@ -107,15 +107,11 @@ async def test_TC_RVCCLEANM_2_2(self):
         self.directmodech_bit_mask = Clusters.RvcCleanMode.Bitmaps.Feature.kDirectModeChange
         self.endpoint = self.matter_test_config.endpoint
         self.is_ci = self.check_pics("PICS_SDK_CI_ONLY")
+        app_pid = self.matter_test_config.app_pid
         if self.is_ci:
             app_pid = self.matter_test_config.app_pid
             if app_pid == 0:
-                asserts.fail("The --app-pid flag must be set when PICS_SDK_CI_ONLY is set.c")
-            self.app_pipe = self.app_pipe + str(app_pid)
```

Comment:
I think that this is needed as `write_to_app_pipe` is used later without specifying the `app_pipe_name`.

Reply:
Thank you, you are indeed correct that was definitely needed in there and not sure how I had deleted it.
However, I have returned it to the code now.